helm/reana/values.yaml

```diff
@@ -60,6 +60,7 @@ components:
     image: reanahub/reana-server:0.7.1
     environment:
       REANA_MAX_CONCURRENT_BATCH_WORKFLOWS: 30
+      REANA_USER_CONFIRMATION: true
```

Comment:
Minor: we are just missing a line about this new variable in the [config docs](https://github.com/reanahub/reana/tree/master/helm/reana#configuration).

Reply:
Forgot to add it as a default value, thanks!